src/v/config/configuration.cc

```diff
@@ -1766,6 +1766,19 @@ configuration::configuration()
       "exception is thrown instead.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       true)
+  , sampled_memory_profile(
+      *this,
+      "memory_enable_memory_sampling",
```

Comment:
So the rate (at least in this change) is not configurable, correct?

Reply:
Correct yes

Reply:
Thanks.